### PR TITLE
ADOP-817 changes for CDAM integration

### PR DESCRIPTION
**Pull request by sairamkantheti1**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link
https://tools.hmcts.net/jira/browse/ADOP-817

**Does this PR introduce a breaking change?** (check one with "x")

- [ ] Yes
- [ ] No

### CODE REVIEW

**Comment by gauravtomar-gh** [comment]
@abidfs, This is going to PROD, Please review.

**Comment by gauravtomar-gh** [comment]
@praneethhm / @sairamkantheti1 , PACT tests are missing for CDAM API. Please add.

File: config/default.yaml [hunk 1]
```diff
@@ -23,6 +23,8 @@ services:
     url: ''
   documentManagement:
     url: ''
+  cdam:
+    url: ''
```
Comment (abidfs) [inline config/default.yaml:27]: In order for this to work from local pointing to aat, you might need to update the endpoint secret in adoption aat keyvault and use it [here](https://github.com/hmcts/adoption-web/blob/6b7447621ac131d863770b2e197a1292555579ee/src/main/modules/properties-volume/index.ts#L57)

**Review by abidfs — CHANGES_REQUESTED** [review]
Looks good to me 
Just a minor logic change to load cdam url from aat keyvault

File: config/default.yaml (hunk 1 above)
Comment (sairamkantheti1) [reply config/default.yaml:27]: I added cdam url to Azure hat endpoints. made changes in index.ts to use that url. It is working fine

**Review by abidfs — DISMISSED** [review]

**Review by pankajhmcts — DISMISSED** [review]

**Review by pankajhmcts — DISMISSED** [review]

**Review by priteshkr — APPROVED** [review]
Approved.Looking good

**Review by praneethhm — APPROVED** [review]

**Pull request merged**